packages/api/tests/endpoints/providers/update-provider.js

```diff
@@ -100,15 +98,20 @@ test.serial('PUT updates an existing provider and returns it in listing', async
     .set('Authorization', `Bearer ${jwtAuthToken}`)
     .expect(200);
 
-  t.plan(1);
-  const response = await request(app)
+  const { body: actualProvider } = await request(app)
     .get(`/providers/${t.context.testProvider.id}`)
     .set('Accept', 'application/json')
     .set('Authorization', `Bearer ${jwtAuthToken}`)
     .expect(200);
-  let results = response.body;
-  results = pick(results, Object.keys(results).filter((key) => !['createdAt', 'updatedAt'].includes(key)));
-  t.deepEqual(results, updatedProvider);
+
+  const expectedProvider = {
+    ...t.context.testProvider,
+    ...updateParams,
+    createdAt: actualProvider.createdAt,
+    updatedAt: actualProvider.updatedAt
+  };
+
+  t.deepEqual(actualProvider, expectedProvider);
```

Comment:
I don't mind inserting instead of removing particularly as it means no lodash.  👍